tdrs-backend/tdpservice/parsers/row_schema.py

```diff
@@ -12,13 +12,13 @@ class RowSchema:
 
     def __init__(
             self,
-            model=dict,
+            document,
             preparsing_validators=[],
             postparsing_validators=[],
             fields=[],
-            quiet_preparser_errors=False
+            quiet_preparser_errors=False,
             ):
-        self.model = model
+        self.document = document
```

Comment:
this change makes me nervous because of the potential side-effects, but you've covered all the cases i've thought of so far